scripts/vpc-destroy-cleanup.sh

```
@@ -0,0 +1,45 @@
+    --output text 2>/dev/null || echo 0)"
+
+  if [[ "$attached_enis" != "0" ]]; then
+    continue
+  fi
+
+  if [[ "$sg_name" == *"-vpce-private-router" ]]; then
+    aws ec2 delete-security-group --group-id "$sg_id" >/dev/null 2>&1 || true
+  fi
+done < <(aws ec2 describe-security-groups \
+  --filters "Name=vpc-id,Values=${VPC_ID}" \
+  --query 'SecurityGroups[?GroupName!=`default`].[GroupId,GroupName,Description]' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)